redis_aamks/manager.py

```diff
@@ -11,7 +11,9 @@
 class RedisManager:
     def __init__(self):
         self.redis_pwd = os.path.join(os.environ['AAMKS_PATH'], "redis_aamks")
+        self.server_path = os.path.join(self.redis_pwd, "worker", "server.py")
         self.worker_path = os.path.join(self.redis_pwd, "worker", "worker.py")
+        self.server_path = os.path.join(self.redis_pwd, "worker", "server.py")
         self.container_name = "aamks_redis"
         self.net_conf = "/etc/aamksconf.json" 
         self.host_array = []
```

Comment:
QueueManago() call should by with try-except. When server is not running QueueManago is not able to connect to redis DB and throws an error preventing any command to run, including help and serverstart (sic!)